docs/guides/developer/docs/infrastructure/index.md

```diff
@@ -67,15 +66,14 @@ include:
 
 - Adding new committers to the relevant group(s)
 - Removing old committers from the relevant group(s)
-- Contacting support when required for hosted projects (Atlassian, Crowdin, Google)
+- Contacting support when required for hosted projects (Crowdin, Google)
```

Comment:
We will still have Confluence and JIRA around for a while though.

Reply:
I suppose.  I feel like there's little change that we would need actual support, but reverting this won't hurt.